server/container/guice/jpa-guice/src/test/resources/mailetcontainer.xml

```diff
@@ -60,7 +60,13 @@
                 <processor>local-address-error</processor>
                 <notice>550 - Requested action not taken: no such user here</notice>
             </mailet>
-            <mailet match="SMTPAuthSuccessful" class="RemoteDelivery">
+            <mailet match="SMTPAuthSuccessful" class="SetMailAttribute">
```

Comment:
Same answer than before. The logic wasn't better before and we add no issue why ? How can I check that a mail is from JMAP ?